project/msvc/faac.pc

```diff
@@ -0,0 +1,9 @@
+prefix=c:/Users/aaron

```

Comment:
This is probably wrong.

Reply:
Yes :)  Better to generate it from a passed-in prefix.

Reply:
I'd prefer this file to get generated from a faad.pc.in template during the `configure` phase, but I don't know how this could be translated to the msvc world.

Reply:
That would be great. I am building on windows with meson, and meson understands .pc files, so this would work. The only question is how to generate the .pc file with visual studio.

Reply:
The windows build of `faac` should somehow generate the .pc file

Reply:
Alternatively, `msys2` on windows could also generate a .pc file from `faac.pc.in`

Reply:
Yes, that's what I was thinking about. But this would need to be prepared first and moved out if the msvc directory. And then we'd still have no solution for msvc.

Reply:
tbh, I'm not sure many people actually use pkg-config on windows. Another solution is to add meson support to the library. Would you be open to that ?  With meson, `faac` could be easily added to GStreamer `gst-build`

Reply:
I guess we could have a solution to all of this by switching the entire build system to cmake.

Reply:
Sure, that would be better than storing a .sln file.